internal/services/compute/image_resource.go

```diff
@@ -372,6 +396,11 @@ func expandImageOSDisk(input []interface{}) *images.ImageOSDisk {
 			}
 		}
 
+		if features.FourPointOhBeta() {
+			if storageAccountType := config["storage_type"].(string); storageAccountType != "" {
+				out.StorageAccountType = pointer.To(images.StorageAccountTypes(storageAccountType))
+			}
```

Comment:
If this is being made a required property, we can set it unconditionally.

```suggestion
			out.StorageAccountType = pointer.To(images.StorageAccountTypes(storageAccountType))
```